src/components/board/board.jsx

```diff
@@ -1,34 +1,40 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { Container, Row } from 'react-bootstrap';
+import PropTypes from 'prop-types';
 import Square from '../square';
 
-export default class Board extends Component {
-  renderSquare(int) {
+
+ const Board = (props) => {
+  function renderSquare(int) {
```

Comment:
This method can be moved outside the component rather to prevent bloat within the component.